plasmapy/formulary/__init__.py

```diff
@@ -16,3 +18,9 @@
 from .quantum import *
 from .relativity import *
 from .ionization import *
+
+# auto populate __all__
+for obj_name in list(globals()):
+    if not (obj_name.startswith("__") or obj_name.endswith("__")):
+        __all__.append(obj_name)
+__all__.sort()
```

Comment:
... huh. Why did this have to happen?

Reply:
You know, I think I might just review purely through LotR soundbites from now on:

https://www.youtube.com/watch?v=J0gubm_mSHk

Reply:
I did this so the [giant tables in `plasmapy.formulary.rst`](https://plasmapy--878.org.readthedocs.build/en/878/api_static/plasmapy.formulary.html#module-plasmapy.formulary) are alphabetized.

Reply:
> You know, I think I might just review purely through LotR soundbites from now on:

https://youtu.be/c3sOuEv0E2I

Reply:
Oooooh. That does make sense. :+1: